ocm-container.sh

```diff
@@ -74,9 +74,19 @@ export OCM_CONTAINER_CONFIGFILE="$CONFIG_DIR/env.source"
 
 if [ ! -f ${OCM_CONTAINER_CONFIGFILE} ]; then
     echo "Cannot find config file at $OCM_CONTAINER_CONFIGFILE";
-    echo "Run the init.sh file to create one."
-    echo "exiting"
-    exit 1;
+    echo "Downloading sample config from upstream..."
+    mkdir -p ${CONFIG_DIR}
+    curl -s https://raw.githubusercontent.com/openshift/ocm-container/master/env.source.sample --output ${OCM_CONTAINER_CONFIGFILE}
+    read -t 300 -p 'Paste your ocm token from https://cloud.redhat.com/openshift/token: ' CONFIG_OCM_TOKEN
```

Comment:
I am thinking it would be good to do this if the configfile exists but the token is still set to `YOUR_TOKEN_HERE`.

Reply:
it wouldn't happen as a result of this workflow, because if the token is not specified in the 5min window, the downloaded file is deleted and ocm container exits 
if the user downloads the config manually and places it there, setting the config is on them

on a side note, if we want to check the validity of the ocm token, we should probably source the config and verify the character length of the `OFFLINE_ACCESS_TOKEN` variable